doc_user/UserManual.adoc

```diff
@@ -828,6 +828,348 @@ image:image251.png[success]
++
+NOTE: This button is enabled just for the product/lot whose column *New Lot* has value *N*, otherwise the button is disabled.
+
+* *[.underline]##D##elete*: used to remove one or more products/lots from the list.
+* *Clean [.underline]##T##able*: used to remove all the inventory rows.
+* *[.underline]##V##alidate*: used to validate the inventory.
+* *[.underline]##C##onfirm*: used to confirm the inventory.
++
+NOTE: The *Confirm* button is enabled once the inventory is validated, until then the button is disabled. See <<inventory-main-characteristics,Main Characteristics>> chapter.
+
+* *[.underline]##C##lose*: used to return to the *_Inventory Browser_* window.
+
+====== 4.4.2.1.1 Insert Inventory Row By Using TextField
+ 
+To insert an inventory row for a medical, type into the *Add medical* field the code or the description of the pharmaceutical and press ENTER. This will open the _Choose a Medical_ window.
+
+image:image254.png[Choose a medical]
+
+Select the desired pharmaceutical or search the pharmaceutical by code or description using the search field and select the pharmaceutical to highlight it, then click on *[.underline]##S##elect* button to add the pharmaceutical as inventory row or click on *[.underline]##C##ancel* to back to the *_Inventory Browser_* window.
+
+====== 4.4.2.1.2 Insert Inventory Row Product By using SELECT Button
+
+To insert an inventory row for a set of medicals, useful for rotational inventories, click on the SELECT button. This will open the Lot Information window.
+
+image:image255.png[Lot Information]
+
+Specify the Medical Type on the combo box (All = all medical types) and choose:
+
+* *All*: if all the pharmaceuticals for the selected Medical Type are needed.
+* *Only non zero quantity*: if only the pharmaceuticals for the selected Medical Type with non zero theoretical quantity are needed.
+* *with movement only*: if only the pharmaceuticals for the selected Medical Type which have been used at least once are needed.
+
+Then click on *[.underline]##O##k* button to insert all the pharmaceuticals that match the choice as the inventory(ies) row(s) in the *Inventory row table*.
+
+===== 4.4.2.2 Edit Inventory ([.underline]##E##dit Inventory)
+
+To modify an inventory, select the inventory in the *_Inventory Browser_* window. Once the inventory has been highlighted, click on *[.underline]##E##dit* button to enter on the *_Edit Inventory_* window. Now the record is available for changes. This function allows editing all the data of the inventory including inventory(ies) row(s) of the inventory.
+
+NOTE: only inventories in DRAFT and VALIDATED states can be edited.
+
+image:image256.png[Edit Inventory]
+
+During modification it is possible to validate the inventory by clicking on the *[.underline]##V##alidate* button. This may results in the following dialog:
```

Comment:
```suggestion
During modification it is possible to validate the inventory by clicking on the *[.underline]##V##alidate* button. This may result in the following dialog:
```